### ISSUE TITLE
Witch Doctor

### ISSUE THREAD

**Issue by Areshar** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth#685]
**DO NOT REMOVE PRE-EXISTING LINES**
----------------------------------------------------------------------------------------------------------
**Describe your suggestion in full detail below:**
Hi there,
I would like to ask if its possible to combine Loa Priest with Necromancer, because it would (atleast at Roleplaying) create something that looks like iconic jungle troll class, which is Witch Doctor. I dont need whole new class... but it seems impossible to have both loa priest and necromancer on one character.

**Comment by zumbak04** [comment]
Loa Priest is based on Witch Doctors (mostly from Wacraft 3). The trolls call every mage a witch doctor even if it's clear Nercomancer like Zanzil the Outcast.

**Comment by Areshar** [comment]
Ok. I cannot say, i agree that trolls see every mage as witch doctor, but i understand, what you mean. My problem with it, is that witch doctors (even good ones) should be able to communicate (possibly raise undead), but in-game i cannot be both Loa Priest and Necromancer at the same time. This combination would provide most sense as witch doctor. Heal and Hex of Loa Priest with Raise undead of Necromancer.

**Comment by zumbak04** [comment]
The most of witch doctors aren't necromancers and can't raise the dead. I know only Zum'rah and Zanzil.
https://wow.gamepedia.com/Zanzil_the_Outcast
https://wow.gamepedia.com/Witch_Doctor_Zum%27rah
And both of them can be clasified as necromancers.

**Comment by Areshar** [comment]
I agree that witch doctors are not necromancers, but they use voodoo. And voodoo is usually shadow/dark magic. One of powers you gain or you could possibly learn to cast is raise the dead. So as you said, many witch doctors dont raise the dead, but i would say, thats by their choice and not by the lack of talent or different magic school.

**Comment by zumbak04** [comment]
Yep, that's their choice to pick Necromancer class. Again, I wouldn't classify them as anything besides necromancer. They raise the dead... and that's all.

**Comment by Areshar** [comment]
I agree. That´s why i would like to have choice ingame to be both loa-priest and necromancer. Because by occasionally raising few dead people, you will not lose your healing and hex powers :-D

**Comment by zumbak04** [comment]
> you will not lose your healing and hex powers :-D

Such witch doctors don't have hex powers and healing. That's the main problem.

**Comment by Areshar** [comment]
There is atleast one that have power of raise the dead and healing [https://wow.gamepedia.com/Witch_Doctor_Zum%27rah](url), even though he doesn´t have hexes is his spellbook... he should be able to cast them in lore. 
There is another that has healing power, hexes and summon shade, which could be counted as "undead or shadow creature" [https://wow.gamepedia.com/Jin%27do_the_Hexxer](url).
For me Witch Doctor is alchemist (potions/elixirs/poisons) and spellcaster that uses usually nature and shadow magic and if someone is telling me that shadow magic is not used for necromancy rituals(not always ofc), than thats mistake on his/her part.

**Comment by zumbak04** [comment]
> There is atleast one that have power of raise the dead and healing [https://wow.gamepedia.com/Witch_Doctor_Zum%27rah](url), even though he doesn´t have hexes is his spellbook... he should be able to cast them in lore.
> There is another that has healing power, hexes and summon shade, which could be counted as "undead or shadow creature" [https://wow.gamepedia.com/Jin%27do_the_Hexxer](url).

Witch Doctors from early WoW expansions have EVERY time of magic in speelbook. So, if we add Loa Priest + Necromancer, there should be Loa Priest + Mage, Loa Priest + Prist, Loa Priest + Shadow Priest, Loa Priest + Warlock. I left links in the message above.

> For me Witch Doctor is alchemist (potions/elixirs/poisons) and spellcaster that uses usually nature and shadow magic and if someone is telling me that shadow magic is not used for necromancy rituals(not always ofc), than thats mistake on his/her part.

Not, it's not.
https://wow.gamepedia.com/Undead
Blizzard said it in the Chronicles.

**Comment by zumbak04** [comment]
Can suggest you one thing: subclass of Necromancer that gives you hex ability. But have no idea how to name it. It's not witch doctor since the most of them have nothing to do with nercomancy.

**Comment by Areshar** [comment]
> Not, it's not.
https://wow.gamepedia.com/Undead
Blizzard said it in the Chronicles.

I dont know that. Thank you for clarification.
 Though i would still think its related, if Light and Void are most fundamental forces in WoW Universe, i would assume that, when Light could resurrect person with Holy magic, Void should do opposite of it.
Even if witch doctors dont use necromantic powers... how is it possible that in [https://wow.gamepedia.com/Voodoo](https://wow.gamepedia.com/Voodoo), there is text, which directly states raising the dead?

**Comment by zumbak04** [comment]
> Though i would still think its related, if Light and Void are most fundamental forces in WoW Universe, i would assume that, when Light could resurrect person with Holy magic, Void should do opposite of it.
> Even if witch doctors dont use necromantic powers... how is it possible that in https://wow.gamepedia.com/Voodoo, there is text, which directly states raising the dead?

**Comment by zumbak04** [comment]
> Even if witch doctors dont use necromantic powers... how is it possible that in https://wow.gamepedia.com/Voodoo, there is text, which directly states raising the dead?

The trolls call everything voodoo and every mage a witch doctor. Shame on them.
